server/package.json

```diff
@@ -32,8 +32,8 @@
     "connect-pg-simple": "^6.2.1",
     "dotenv": "^8.2.0",
     "dotenv-cli": "^4.0.0",
-    "express": "^4.17.1",
-    "express-session": "^1.17.1",
+    "express": "^5.0.0-alpha.8",
```

Comment:
I wanted to jump to it since it improves the error handling when promises fail. You know how Express just hangs when an error occurs - this makes it so Express will return a 500 immediately, and it also makes it so you can see the full error stack trace instead of just the error message.

Reply:
Understood, thank you for the clarification there!